UniWatch/Controllers/ClassController.cs

```diff
@@ -88,7 +87,7 @@ public ActionResult Create([Bind(Include = "Name,Number,Section,Semester,Year")]
         public ActionResult Delete(int classId)
```

Comment:
Authorization attribute is controller wide. Only a teacher can get here. Was tested.